Makefile.am

```diff
@@ -1,3 +1,4 @@
+ACLOCAL_AMFLAGS = -I m4
```

Comment:
What is occurring with this not set?

Reply:
This fixes `libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.`.